MediaBrowser.Providers/Plugins/Tmdb/TV/TmdbEpisodeProvider.cs

```diff
@@ -92,7 +92,7 @@ public async Task<MetadataResult<Episode>> GetMetadata(EpisodeInfo info, Cancell
             }
 
             var episodeResult = await _tmdbClientManager
-                .GetEpisodeAsync(seriesTmdbId, seasonNumber.Value, episodeNumber.Value, info.SeriesDisplayOrder, info.MetadataLanguage, TmdbUtils.GetImageLanguagesParam(info.MetadataLanguage), cancellationToken)
+                .GetEpisodeAsync(seriesTmdbId, seasonNumber.Value, episodeNumber.Value, !string.IsNullOrEmpty(info.TmdbEpisodeGroupId) ? info.TmdbEpisodeGroupId : string.Empty, info.MetadataLanguage, TmdbUtils.GetImageLanguagesParam(info.MetadataLanguage), cancellationToken)
```

Comment:
Instead of slapping a ternary statement as a parameter to essentially retrieve the value, we can simply add
```suggestion
                .GetEpisodeAsync(seriesTmdbId, seasonNumber.Value, episodeNumber.Value, info?.TmdbEpisodeGroupId ?? string.Empty, info.MetadataLanguage, TmdbUtils.GetImageLanguagesParam(info.MetadataLanguage), cancellationToken)
```

This way we won't deal with a null reference exception in the scenario that TmdbEpisodeGroupId turns up null for any reason